FhirToDataLake/SynapseValidate/Synapse_Validator.py

```diff
@@ -0,0 +1,207 @@
+import os
+import sys
+import argparse
+import concurrent.futures
+import json
+import requests
+import pyodbc
+import pandas as pd
+import time
+from urllib import parse as urlparse
+from datetime import datetime
+
+parser = argparse.ArgumentParser()
+parser.add_argument("--SynapseWorkspaceName", help="Name of Synapse workspace.")
+parser.add_argument("--FhirServerUrl", help="Fhir server url.")
+parser.add_argument("--Database", help="Name of database.")
+parser.add_argument("--SchemaCollectionDirectory", help="Schema collection directory path.")
+args = parser.parse_args()
+
+Sql_Server_Endpoint = args.SynapseWorkspaceName + "-ondemand.sql.azuresynapse.net"
+Database = args.Database
+Schema_Collection_Directory = args.SchemaCollectionDirectory
+Fhir_Server_Base_Url = args.FhirServerUrl
+Sql_Username = os.getenv('SQL_USERNAME')
+Sql_Password = os.getenv('SQL_PASSWORD')
+
+pyodbc.pooling = False
+
+class RawDataSource:
+    Headers = {'Accept': 'application/fhir+json', "Prefer": "respond-async"}
+    Start_Datetime = "1970-01-01T00:00:00-00:00"
```

Comment:
Nit: It would be better to name variables in `CamelCase` (StartDateTime) or `snack_case` (start_date_time) for better readability. So do other variables.

Reply:
Thanks, have checked and fixed name in this script. Now have renamed to **SynapseValidator.py**